addons/l10n_cz/data/template/account.account-cz.csv

```diff
@@ -1,270 +1,270 @@
-"chart_cz_591000","Daň z příjmů splatná","591000","expense","False"
-"chart_cz_592000","Daň z příjmů odložená","592000","expense","False"
-"chart_cz_593000","Daň z příjmů z mimořádné činnosti - splatná","593000","expense","False"
-"chart_cz_595000","Dodatečné odvody daně z příjmů","595000","expense","False"
-"chart_cz_596000","Převod podílu na výsledku hospodaření společníkům v.o.s. a komplementářům k.s.","596000","expense","False"
-"chart_cz_597000","Převod provozních nákladů","597000","expense","False"
-"chart_cz_598000","Převod finančních nákladů","598000","expense","False"
-"chart_cz_599000","Změna stavu rezervy na daň z příjmů","599000","expense","False"
-"chart_cz_601000","Tržby za vlastní výrobky","601000","income","False"
-"chart_cz_602000","Tržby z prodeje služeb","602000","income","False"
-"chart_cz_604000","Tržby za zboží","604000","income","False"
-"chart_cz_641000","Tržby z prodeje dlouhodobého nehmotného a hmotného majetku","641000","income","False"
-"chart_cz_642000","Tržby z prodeje materiálu","642000","income","False"
-"chart_cz_643000","Přijaté dary v provozní oblasti","643000","income","False"
-"chart_cz_644000","Smluvní pokuty a úroky z prodlení","644000","income","False"
-"chart_cz_645000","Výnosy z postoupených pohledávek","645000","income","False"
-"chart_cz_646000","Výnosy z odepsaných pohledávek","646000","income","False"
-"chart_cz_647000","Mimořádné provozní výnosy","647000","income","False"
-"chart_cz_648000","Ostatní provozní výnosy","648000","income","False"
-"chart_cz_649000","Odpis záporného goodwillu a zúčtování oprávky k oceň. rozdílu k nabytému majetku","649000","income","False"
-"chart_cz_661000","Tržby z prodeje cenných papírů a podílů","661000","income","False"
-"chart_cz_662000","Úroky","662000","income","False"
-"chart_cz_663000","Kurzové zisky","663000","income","False"
-"chart_cz_664000","Výnosy z přecenění cenných papírů","664000","income","False"
-"chart_cz_665000","Výnosy z dlouhodobého finančního majetku","665000","income","False"
-"chart_cz_666000","Výnosy z krátkodobého finančního majetku","666000","income","False"
-"chart_cz_667000","Výnosy z derivátových operací","667000","income","False"
-"chart_cz_668000","Ostatní finanční výnosy","668000","income","False"
-"chart_cz_669000","Přijaté dary ve finanční oblasti","669000","income","False"
-"chart_cz_697000","Převod provozních výnosů","697000","income","False"
-"chart_cz_698000","Převod finančních výnosů","698000","income","False"
-"chart_cz_699000","Výnosy hospodářských středisek","699000","income","False"
-"chart_cz_701000","Počáteční účet rozvažný","701000","off_balance","False"
-"chart_cz_702000","Konečný účet rozvažný","702000","off_balance","False"
-"chart_cz_710000","Účet zisků a ztrát","710000","off_balance","False"
+"id","name","code","account_type","reconcile","name@cz"
+"chart_cz_012000","Intangible results of research and development","12000","asset_non_current","False","Nehmotné výsledky výzkumu a vývoje"
```

Comment:
i would keep the 0 in front to keep the consistency with other loca where the id number is the same as the code.

It happens multiple times, i will just put the msg once :smile: